<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cconsole</class>
 <widget class="QDialog" name="Cconsole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cconsole</string>
  </property>
  <widget class="QDial" name="dial">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ZongEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>纵向速度：</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>161</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QLineEdit" name="PixelEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>曝光：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>增益</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>比例</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="ExposeEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="GainEdit"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="WidthEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>线宽：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDial" name="dial_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>横向速度：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="HengEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>吸气</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>保存并退出</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>纵向启动</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>横向启动</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>dial</zorder>
  <zorder>ZongEdit</zorder>
  <zorder>label</zorder>
  <zorder>gridLayoutWidget</zorder>
  <zorder>dial_2</zorder>
  <zorder>label_6</zorder>
  <zorder>gridLayoutWidget_2</zorder>
  <zorder>HengEdit</zorder>
  <zorder>label_7</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButton_2</tabstop>
  <tabstop>ZongEdit</tabstop>
  <tabstop>HengEdit</tabstop>
  <tabstop>ExposeEdit</tabstop>
  <tabstop>GainEdit</tabstop>
  <tabstop>PixelEdit</tabstop>
  <tabstop>WidthEdit</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>dial</tabstop>
  <tabstop>dial_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Cconsole</receiver>
   <slot>returnmain()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Cconsole</receiver>
   <slot>Dialnum_1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Cconsole</receiver>
   <slot>Dialnum_2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Cconsole</receiver>
   <slot>horizontal_speed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Cconsole</receiver>
   <slot>vertical_speed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Cconsole</receiver>
   <slot>testqi()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>24</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Cconsole</receiver>
   <slot>save_quit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>returnmain()</slot>
  <slot>Dialnum_1()</slot>
  <slot>Dialnum_2()</slot>
  <slot>vertical_speed()</slot>
  <slot>horizontal_speed()</slot>
  <slot>save_quit()</slot>
  <slot>testqi()</slot>
 </slots>
</ui>
